httpserver/rust/src/httpserver.rs

```diff
@@ -271,6 +278,10 @@ pub struct HttpResponse {
     #[serde(with = "serde_bytes")]
     #[serde(default)]
     pub body: Vec<u8>,
+    /// id to mention whether to keep the websocket client active or not for the future. Maybe an empty string.
+    #[serde(rename = "persistWebSocketClientWithThisID")]
+    #[serde(default)]
+    pub persist_websocket_client_with_id: String
```

Comment:
When you persist a websocket client, would this be in the goal of sending future frames to the exact same actor?